app/src/main/java/com/github/fribourgsdp/radio/MusixmatchLyricsGetter.kt

```diff
@@ -32,147 +30,173 @@ interface LyricsGetter{
  * API Call doc : https://stackoverflow.com/questions/45219379/how-to-make-an-api-request-in-kotlin
  */
 object MusixmatchLyricsGetter : LyricsGetter {
-        const val LYRICS_NOT_FOUND = "---No lyrics were found for this song.---"
+    const val LYRICS_NOT_FOUND = "---No lyrics were found for this song. Feel free to add yours !---"
+    const val BACKEND_ERROR_PLACEHOLDER = ""
 
-        class LyricsNotFoundException(val reason : String = LYRICS_NOT_FOUND) : Exception()
+    abstract class LyricsGetterException(val default : String) : Exception()
+    class NoLyricsFoundForThisSong : LyricsGetterException(LYRICS_NOT_FOUND)
+    class BackendError : LyricsGetterException("")
```

Comment:
Here you could use the BACKEND_ERROR_PLACEHOLDER you defined above.

Reply:
yep !